### ISSUE TITLE
[Enhancement] Add new orange color to replace the existing warning notification color

### ISSUE THREAD

**Issue by Fuzzy3** [kirbydesign/designsystem#3968]
The currently used warning notification color is not contrasty enough in the eyes of evil mr wcag so we need to introduce the brand new super hero color orangy-boy. Not only does he shine bright like a diamond, he's also the hero we don't deserve. 

  
## Describe the enhancement
His cape takes many colors and here's the complete list:

| Variable | Color  | Visual |
| -------- | ------ | ------ |
| kirby-decoration-color-orange-10  | #FFEAD8 | ![Image](https://github.com/user-attachments/assets/a3132cd1-073f-483c-8a62-d2ae87241a2d)  |
| kirby-decoration-color-orange-20 | #FFD8B5 | ![Image](https://github.com/user-attachments/assets/52cb5872-4a87-4532-a532-1c8444c71a9a) |
| kirby-decoration-color-orange-30  | #FFBB81  |   ![Image](https://github.com/user-attachments/assets/eff261db-c459-4a46-b5fc-5cfc90ddfb8c) |
| kirby-decoration-color-orange-40 | #FFAA61 | ![Image](https://github.com/user-attachments/assets/f46127ff-217e-4d0d-af18-5fb6b47c2ef9) |
| kirby-decoration-color-orange-50 | #FF9D48 | ![Image](https://github.com/user-attachments/assets/e932278d-c678-4539-918d-4d38aa04449e) |
| kirby-decoration-color-orange-60 | #D9731B | ![Image](https://github.com/user-attachments/assets/fe0f8e57-38ab-457f-9e41-5ffaaf3be6a0) |
| kirby-decoration-color-orange-70 | #B56118 | ![Image](https://github.com/user-attachments/assets/f5aa9eae-699a-46cd-9461-e6b99af69291) |
| kirby-decoration-color-orange-80 | #8E511B | ![Image](https://github.com/user-attachments/assets/0c05e33b-5833-4008-a473-8214b0bbd99a) |
| kirby-decoration-color-orange-90 | #5A310D | ![Image](https://github.com/user-attachments/assets/dd781da7-557f-4624-885d-f134e882a031) |

## Describe the solution you'd like
When the orange color has been introduced, the warning notification color must point to this instead of the "old" yellow

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".
